diff --git a/models/aws_robomaker_warehouse_GroundB_01/model.sdf b/models/aws_robomaker_warehouse_GroundB_01/model.sdf
index 7d5a116..0213ef0 100644
--- a/models/aws_robomaker_warehouse_GroundB_01/model.sdf
+++ b/models/aws_robomaker_warehouse_GroundB_01/model.sdf
@@ -23,8 +23,8 @@
         <surface>
           <friction>
             <ode>
-              <mu>0.3</mu>
-              <mu2>0.3</mu2>
+              <mu>0.7</mu>
+              <mu2>0.7</mu2>
               <fdir1>0 0 0</fdir1>
               <slip1>0</slip1>
               <slip2>0</slip2>
